docs/src/pages/components/pagination/PaginationOutlined.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Pagination from '@material-ui/lab/Pagination';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    '& > *': {
```

Comment:
I hope that we figure styling out in the next 10 years. After that we'll look at this syntax and wonder how we ever thought this was ok :smile:

Reply:
I wonder that now. 😁